boards/hail/src/main.rs

```diff
@@ -53,12 +54,13 @@ struct Hail {
     console: &'static Console<'static, usart::USART>,
     gpio: &'static capsules::gpio::GPIO<'static, sam4l::gpio::GPIOPin>,
     timer: &'static TimerDriver<'static, VirtualMuxAlarm<'static, sam4l::ast::Ast<'static>>>,
-    isl29035: &'static capsules::isl29035::Isl29035<'static,
-                                                    VirtualMuxAlarm<'static,
-                                                                    sam4l::ast::Ast<'static>>>,
-    si7021: &'static capsules::si7021::SI7021<'static,
-                                              VirtualMuxAlarm<'static, sam4l::ast::Ast<'static>>>,
-    ninedof: &'static capsules::ninedof::NineDof<'static>,
+    //isl29035: &'static capsules::isl29035::Isl29035<'static,
+    //                                                VirtualMuxAlarm<'static,
+    //                                                                sam4l::ast::Ast<'static>>>,
+    //si7021: &'static capsules::si7021::SI7021<'static,
+    //                                          VirtualMuxAlarm<'static, sam4l::ast::Ast<'static>>>,
+    //ninedof: &'static capsules::ninedof::NineDof<'static>,
+    //i2c_master_slave: &'static capsules::i2c_master_slave_driver::I2CMasterSlaveDriver<'static>,
```

Comment:
For now, I guess my recommendation would be to bring back everything on the main I2C bus, convert D2-5 to I2C mode, use D2/3 for Mod In/Out, and use TWIMS0 for the Signbus I2C bus.